modules/mobkoiIdSystem.js

```diff
@@ -93,14 +93,14 @@ submodule('userId', mobkoiIdSubmodule);
 export const utils = {
   requestEquativSasId(syncUserOptions, consentObject, onCompleteCallback) {
     logInfo('Start requesting Equativ SAS ID');
-    const adServerBaseUrl = deepAccess(
+    const integrationBaseUrl = deepAccess(
       syncUserOptions,
-      `params.${PARAM_NAME_AD_SERVER_BASE_URL}`) || PROD_AD_SERVER_BASE_URL;
+      `params.${PARAM_NAME_INTEGRATION_BASE_URL}`) || PROD_INTEGRATION_BASE_URL;
 
     const equativPixelUrl = utils.buildEquativPixelUrl(syncUserOptions, consentObject);
     logInfo('Equativ SAS ID request URL:', equativPixelUrl);
 
-    const url = adServerBaseUrl + '/pixeliframe?' +
+    const url = integrationBaseUrl + '/pixeliframe?' +
```

Comment:
MOBKOI_PIXEL_SYNC_ERROR and MOBKOI_PIXEL_SYNC_COMPLETE below should remove the listener